### PR TITLE
Bump jQuery version, expose jquery globally so it doesn't need to be loaded separately

### PR DESCRIPTION
**Pull request by bookernath**
#### What?

Bump jQuery version to [address a CVE](https://blog.jquery.com/2020/04/10/jquery-3-5-0-released/). I've confirmed the main change (related to [htmlPrefilter()](https://api.jquery.com/jQuery.htmlPrefilter/)) does not affect us.

I also exposed `$` on the window object - because I see that lots of custom themes feel the need to separately include jQuery, which will be bad for performance. Exposing our existing bundle jQuery gets rid of the need to do this.

image

### CODE REVIEW

File: webpack.common.js [hunk 1]
```diff
@@ -36,6 +36,13 @@ module.exports = {
                     },
                 },
             },
+            {
+                test: require.resolve('jquery'),
+                use: [{
+                    loader: 'expose-loader',
+                    options: '$',
```
Comment (junedkazi) [inline webpack.common.js:43]: should we also expose `jQuery` as an option ?

Comment (junedkazi) [reply webpack.common.js:43]: Something like this 
```
{
    test: require.resolve('jquery'),
    use: [{
      loader: 'expose-loader',
      options: 'jQuery'
    },{
      loader: 'expose-loader',
      options: '$'
    }]
  }
```

Comment (bookernath) [reply webpack.common.js:43]: Sure!

Comment (bookernath) [reply webpack.common.js:43]: Couldn't get it working, will fix in a follow up.

**Pull request merged**